netci.groovy

```diff
@@ -198,6 +198,27 @@ commitPullList.each { isPr ->
   addRoslynJob(myJob, jobName, branchName, isPr, triggerPhraseExtra, triggerPhraseOnly)
 }
 
+// Open Integration Tests
+commitPullList.each { isPr ->
+  def jobName = Utilities.getFullJobName(projectName, "open-vsi", isPr)
+  def myJob = job(jobName) {
+    description('open integration tests')
+    label('auto-win2012-20160912')
+    steps {
+      batchFile("""set TEMP=%WORKSPACE%\\Binaries\\Temp
+mkdir %TEMP%
+set TMP=%TEMP%
```

Comment:
Yes definitely a separate Pr. 

---
In reply to: [78596261](https://github.com/dotnet/roslyn/pull/13732#discussion_r78596261) [](ancestors = 78596261)